middleware/proxy/proxy.go

```diff
@@ -43,7 +43,8 @@ type UpstreamHost struct {
 	Fails             int32
 	FailTimeout       time.Duration
 	Unhealthy         bool
-	ExtraHeaders      http.Header
+	UpStreamHeaders   http.Header
+	DownStreamHeaders http.Header
```

Comment:
For some reason my OCD is kicking in and this is bothering me, but can we change "UpStreamHeaders" and "DownStreamHeaders" to "UpstreamHeaders" and "DownstreamHeaders"? Upstream and downstream are one word, and this change will make it consistent with `upstreamHeaders` and `downstreamHeaders` as used below.